api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -262,7 +262,19 @@ message ProviderWithAudiences {
 //          - provider_name: provider-B
 //          - provider_name: provider-C
 //
-// [#next-free-field: 6]
+//  # Example 7: A is optional (if token from A is provided, it must be valid, but also allows missing token.)
+//  requirements:
+//  - provider_name: provider-A
+//    allow_missing: true
+//
+//  # Example 8: A is optional and B is required.
+//  requires_all:
```

Comment:
can you update this example to reflect last change?

Reply:
Done